src/main/java/com/vackosar/gitflowincrementalbuild/boundary/Configuration.java

```diff
@@ -32,6 +35,10 @@
 
 public class Configuration {
 
+    static final String PLUGIN_KEY = "com.vackosar.gitflowincrementalbuilder:gitflow-incremental-builder";
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/gitflow-incremental-builder/gitflow-incremental-builder/pullRequest?prid=5852017)